autogpt_platform/backend/backend/notifications/queue.py

```
@@ -0,0 +1,109 @@
+
+        queue = await get_async_notification_queue()
+        await queue.publish_message(
+            routing_key=routing_key,
+            message=message,
+            exchange=NOTIFICATION_EXCHANGE,
+        )
+        return NotificationResult(
+            success=True, message=f"Queued with routing key: {routing_key}"
+        )
+    except Exception as e:
+        logger.exception(f"Error queueing {routing_key}: {e}")
```

Ruff (0.16.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)